PC/winreg.c

```diff
@@ -1025,8 +1029,13 @@ winreg_DeleteKeyEx_impl(PyObject *module, HKEY key,
                     (Py_ssize_t)access) < 0) {
         return NULL;
     }
-    /* Only available on 64bit platforms, so we must load it
-       dynamically. */
+    /* RegDeleteKeyExW is also available on 32-bit systems.
+     * On such systems, RegDeleteKeyExW() is functionally equivalent to
+     * RegDeleteKeyW().
+     *
+     * Note: Though it currently isn't necessary, this can be simplified to use
+     * load-time dynamic linking in the future.
+     * */
```

Comment:
```suggestion
    /* XXX: remove dynamic load, as RegDeleteKeyExW is always available */
```

Reply:
If you'd like to also remove the dynamic load and make it just call `RegDeleteKeyExW` directly, that would be fine too. Then this comment can go away entirely.